<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>形象进度</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>形象进度文字汇总</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>开始日期：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="start_date">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>结束日期：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="end_date">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>生成进度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>T梁预制：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="beam_casted">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>T梁安装：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="beam_ins">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>湿接缝：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="seam">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>防撞护栏：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="guardrail">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>桥面铺装：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="formation">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionfeedback"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>打开数据库</string>
    </property>
    <addaction name="actionopen_database"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionfeedback">
   <property name="text">
    <string>direction for use</string>
   </property>
  </action>
  <action name="actionopen_database">
   <property name="text">
    <string>open database</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>start_date</tabstop>
  <tabstop>end_date</tabstop>
  <tabstop>beam_casted</tabstop>
  <tabstop>beam_ins</tabstop>
  <tabstop>seam</tabstop>
  <tabstop>guardrail</tabstop>
  <tabstop>formation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
